ansible/roles/fanout/templates/sonic_deploy_202012.j2

```diff
@@ -102,31 +102,6 @@
     {% endfor %}
     },
 
-    "BUFFER_QUEUE": {
```

Comment:
Maybe we need to keep these configurations for platforms which support QoS. I'll add some condition to check that.

Reply:
As discussed offline, these configurations are not mandatory and will be removed in this PR.